course_info/icommons.py

```diff
@@ -122,7 +121,7 @@ def _course_info_instructor_list(self, course_instance_id, **kwargs):
                 instructors = []
                 for person in rv['results']:
                     course_person_schema(person)
-                    if person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
+                    if person.get('role') and person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
```

Comment:
Nope, gave that a shot as well. Gives the following error:
'NoneType' object has no attribute 'get'.

So the issue is when role is null and we return an empty dictionary, can't find a key role_id in it.